src/lxc/cgroups/cgfsng.c

```diff
@@ -1124,6 +1124,60 @@ __cgfsng_ops__ static void cgfsng_payload_destroy(struct cgroup_ops *ops,
 	}
 }
 
+__cgfsng_ops static void cgfsng_monitor_destroy(struct cgroup_ops *ops,
+						struct lxc_handler *handler)
+{
+	int len;
+	char *pivot_path;
+	struct lxc_conf *conf = handler->conf;
+	char pidstr[25];
+
+	if (!ops->hierarchies)
+		return;
+
+	len = snprintf(pidstr, 25, "%d", handler->monitor_pid);
```

Comment:
I'd prefer `sizeof(pidstr)` in all the code paths below. Magic numbers always makes me search for context.